example/android/app/build.gradle

```diff
@@ -386,6 +386,7 @@ dependencies {
   implementation "io.primer:3ds-android:1.2.0"
   implementation "io.primer:ipay88-my-android:1.0.0"
   implementation "io.primer:klarna-android:1.0.1"
+  implementation "io.primer:nol-pay-android:1.0.1"
```

Comment:
Is there a need to include this dependency?

Reply:
Hey @semirp when using `io.primer:android:2.19.1` on calling `PrimerHeadlessUniversalCheckout.current.start()` it was throwing a Failed Resolution of:Lio/primer/nolpay/api/PrimerNolPay when the `io.primer:nol-pay-android:1.0.1` dependency was not included in the `build.gradle`

Reply:
I've now removed the Nol dependency and updated to the latest Android SDK version (2.19.2)